conan/internal/model/conf.py

```diff
@@ -133,6 +133,16 @@
     "tools.build:linker_scripts": "List of linker script files to pass to the linker used by different toolchains like CMakeToolchain, AutotoolsToolchain, and MesonToolchain",
     # Package ID composition
     "tools.info.package_id:confs": "List of existing configuration to be part of the package ID",
+    # Runners
```

Comment:
Here we need @memsharded 's approval. I don't know if we want to add the config to the builtins, because the runners are going to be extra things that will need to install their dependencies separately.

Reply:
Yes, definitely, I added them as extra documentation but they should probably be moved somewhere else or just removed.